hera_mc/cm_hookup.py

```diff
@@ -634,8 +634,8 @@ def show_hookup(self, hookup_dict, cols_to_show='all', state='full', ports=False
         ports:  boolean to include ports or not
         revs:  boolean to include revisions letter or not
         state:  show the full hookups only, or all
-        file:  file to use, None goes to stdout
-        output_format:  set to html for the web-page version, or ascii
+        filename:  file name to use, None goes to stdout
+        output_format:  set to html for the web-page version, or table, or csv
```

Comment:
what does the `table` format mean for file output?

Reply:
It writes the formatted table to that file.

Reply:
so it's a human-readable text file? Something explaining that needs to be in the docstring.